src/invariant/test/ConservationOfLumensTests.cpp

```diff
@@ -80,11 +80,26 @@ updateBalances(std::vector<LedgerEntry> entries, Application& app,
     REQUIRE(initialCoins + netChange == finalCoins);
 
     LedgerTxn ltx(app.getLedgerTxnRoot());
-    ltx.loadHeader().current().totalCoins += netChange;
+    auto& current = ltx.loadHeader().current();
```

Comment:
see my other comment: if we have UB here that means the overall data is just bad.
`totalCoins < 0` indicates that there is something really wrong

Reply:
I've added a `REQUIRE()` to that effect and simplified the rest of the test correspondingly.